lib/exsoda/http.ex

```diff
@@ -108,6 +108,18 @@ defmodule Exsoda.Http do
     end
   end
 
+  defp random_request_id(length \\ 32) do
+    alphabet = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+    numbers = "0123456789"
+    valid = Enum.join([alphabet, String.downcase(alphabet), numbers])
+
+    {_, req_id} = Enum.map_reduce(1..length, "", fn count, req_id ->
+      letter = String.at(valid, :rand.uniform(String.length(valid) - 1))
+      {count..length, Enum.join([req_id, letter])}
+    end)
+    req_id
+  end
+
```

Comment:
Do you think this makes sense? I think it might be a bit easier to read (?)

Also, If we do the charactery stuff at compile time, we don't need to do it on each request, which is kind of nice (but doesn't really matter because it's about to make an http request which will take a while anyway)

```elixir
@alphabet Enum.split("ABCDEFGHIJKLMNOPQRSTUVWXYZ", "")
@numbers Enum.split("0123456789", "")
@valid (@alphabet ++ @numbers ++ Enum.map(@alphabet, &String.downcase/1))

defp random_request_id(len \\ 32) do
  1..len
  |> Enum.map(fn _ -> Enum.random(@valid) end)
  |> Enum.join("")
end
```